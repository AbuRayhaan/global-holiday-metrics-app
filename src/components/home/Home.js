import React from 'react';
import Form from './Form';
import HomeCity from './DefaultCity';

const Home = () => {
  return (
    <section className="">
      <HomeCity />
      <Form />
    </section>
  )
}

export default Home;