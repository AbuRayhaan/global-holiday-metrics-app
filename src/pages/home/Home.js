import React from 'react';
import CountryList from '../../components/home/CountryList';
import GlobalHoliday from '../../components/home/GlobalHoliday';

const Home = () => (
  <section className="">
    <GlobalHoliday />
    <CountryList />
  </section>
);

export default Home;
