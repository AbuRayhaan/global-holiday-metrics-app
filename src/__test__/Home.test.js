import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Home from '../components/home/CountryList';

const HomeStoreProvider = () => (
  <Provider store={store}>
    <Home />
  </Provider>
);

describe('Home component test', () => {
  it('renders correctly', () => {
    const RenderHomeComponent = renderer.create(<HomeStoreProvider />).toJSON();
    expect(RenderHomeComponent).toMatchSnapshot();
  });
});
