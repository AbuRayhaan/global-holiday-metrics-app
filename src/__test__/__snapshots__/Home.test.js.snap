// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component test renders correctly 1`] = `
<div
  className="country-container"
>
  <section>
    <div
      className="form-container d-flex"
    >
      <form>
        <input
          onChange={[Function]}
          placeholder="Search Country"
          type="text"
        />
      </form>
    </div>
    <ul
      className="country-list"
    />
  </section>
</div>
`;
